woocommerce/woocommerce-gateway-stripe
# Implement ECE for payment processing on block checkout page

Fixes #3403  #3408

## Changes proposed in this Pull Request:
- Created hooks, event-handlers and utils functions for ECE in blocks.
- The changes are made following the functions for [payment-request](https://github.com/woocommerce/woocommerce-gateway-stripe/tree/develop/client/blocks/payment-request) and WooPayments. 

## Testing instructions
- Enable the ECE feature flag (return true from `is_stripe_ece_enabled` function or in your database add an option `_wcstripe_feature_ece` and set yes as the value).
- As a shopper add a product to your cart and go to the Block checkout page.
- Confirm that the Google Pay and Apple Pay buttons are loaded.
- Confirm that you can successfully place an order with Google Pay and Apple Pay.
- As a shopper add a product to your cart and go to the Block cart page.
- Confirm that the Google Pay and Apple Pay buttons are loaded.
- Confirm that you can successfully place an order with Google Pay and Apple Pay.
- Disable the feature flag and confirm that PRBs are still working without any regression.
